src/NServiceBus.RavenDB.AcceptanceTests/ConfigureEndpointRavenDBPersistence.cs

```diff
@@ -51,11 +51,7 @@ public static DocumentStore GetDocumentStore()
 
     internal static DocumentStore GetInitializedDocumentStore(string defaultDatabase)
     {
-        var urls = Environment.GetEnvironmentVariable("CommaSeparatedRavenClusterUrls");
-        if (urls == null)
-        {
-            throw new Exception("RavenDB cluster URLs must be specified in an environment variable named CommaSeparatedRavenClusterUrls.");
-        }
+        var urls = Environment.GetEnvironmentVariable("CommaSeparatedRavenClusterUrls") ?? "http://localhost:8080";
```

Comment:
This needs to be rolled back

Reply:
my bad

Reply:
I extracted it into a dedicated PR anyway: https://github.com/Particular/NServiceBus.RavenDB/pull/488